packages/insight/src/utilities/helper-methods.ts

```diff
@@ -187,4 +188,8 @@ export const getDifficultyFromBits = (bits: number) => {
   const maxBody = Math.log(0x00ffff);
   const scaland = Math.log(256);
   return Math.exp(maxBody - Math.log(bits & 0x00ffffff) + scaland * (0x1d - ((bits & 0xff000000) >> 24)));
+}
+
+export const isMobile = () => {
```

Comment:
Right now this function is only fired when a component is rendered and does not watch for `window.innerWidth` change. If you wanted a more robust solution you could do something like:
```
useEffect(() => {
    const isMobile = () => window.innerWidth < Number(size.mobileL.slice(0, -2));
    window.addEventListener('resize', isMobile);
    isMobile();
    return () => window.removeEventListener('resize', isMobile);
  }, [])
  ```
  
  Though, I think it's best to just utilize the existing pattern in the project with native media queries like:
  ```
  @media screen and (max-width: ${size.size.mobileL}) {}
  ```